netdeploy/remote/deployedNetwork.go

```diff
@@ -479,7 +480,8 @@ func (cfg DeployedNetwork) GenerateDatabaseFiles(fileCfgs BootstrappedNetwork, g
 	l.Close()
 
 	localCfg.CatchpointTracking = 0
-	l, err = ledger.OpenLedger(log, genesisFolder+"/bootstrapped", false, initState, localCfg)
+	prefix2 := genesisFolder + "/bootstrapped"
```

Comment:
I realize this is unrelated to your change, but this appears to be the same thing as `prefix`.

Reply:
well, it's my change in the sense that I made `prefix` a thing, where it had been just inlined before. It's concerning that we open the ledger twice with the same path, constructed two different ways. When I noticed it was a second call, and it *wasn't* identically made, I figured I'd just fork them into prefix and prefix2.

I can flip it to just use prefix, defined once.